data/mock-11/dataset-metadata.tsv

```diff
@@ -1,7 +1,9 @@
 name	value
 citation	doi:10.1038/nmeth.2276
 qiita-id	1517
-raw-data-url	ftp://ftp.microbio.me/pub/illumina-mock-communities-raw-data/L18S-1/
+raw-data-url-forward-read	https://dl.dropboxusercontent.com/u/2868868/data/mockrobiota/mock-11/mock-forward-read.fastq.gz
+raw-data-url-reverse-read	https://dl.dropboxusercontent.com/u/2868868/data/mockrobiota/mock-11/mock-reverse-read.fastq.gz
```

Comment:
AH, you're right. I was over-zealous with copy/paste.

On Thu, Sep 8, 2016 at 6:04 PM, Greg Caporaso <notifications@github.com>
wrote:

> In data/mock-11/dataset-metadata.tsv
> <https://github.com/caporaso-lab/mockrobiota/pull/43#discussion_r78105060>
> :
>
> > @@ -1,7 +1,9 @@
> >  name	value
> >  citation	doi:10.1038/nmeth.2276
> >  qiita-id	1517
> > -raw-data-url	ftp://ftp.microbio.me/pub/illumina-mock-communities-raw-data/L18S-1/
> > +raw-data-url-forward-read	https://dl.dropboxusercontent.com/u/2868868/data/mockrobiota/mock-11/mock-forward-read.fastq.gz
> > +raw-data-url-reverse-read	https://dl.dropboxusercontent.com/u/2868868/data/mockrobiota/mock-11/mock-reverse-read.fastq.gz
>
> Do these reverse reads exist? I don't have them in dropbox.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/caporaso-lab/mockrobiota/pull/43/files/dd1115586b170b1ad104ef09202bf0470680472e#r78105060>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB0bbEB_5dhCCzDGix4y9gLz2M1cHfS2ks5qoJR4gaJpZM4J4c-L>
> .
>